src/lib/classes/lsp8-identifiable-digtal-asset.spec.ts

```diff
@@ -74,13 +71,14 @@ describe('LSP8IdentifiableDigitalAsset', () => {
           lsp8Address = deploymentEvent.receipt.contractAddress;
         }
       },
-      error: () => {
-        done();
+      error: (error) => {
+        // Fail to exit subsciber
+        expect(1).toEqual(error);
```

Comment:
maybe this is better?

https://stackoverflow.com/a/59429435/651299

`done.fail()`;